src/libdrakvuf/win-processes.c

```diff
@@ -153,8 +153,74 @@ addr_t win_get_current_process(drakvuf_t drakvuf, uint64_t vcpu_id)
     return process;
 }
 
+static unicode_string_t* convert_to_utf8(unicode_string_t* us)
```

Comment:
You could just use `drakvuf_read_unicode`

Reply:
Ok, I'll use it